inc/thorvg_capi.h

```diff
@@ -579,15 +579,27 @@ TVG_EXPORT Tvg_Paint* tvg_paint_duplicate(Tvg_Paint* paint);
 TVG_EXPORT Tvg_Result tvg_paint_get_bounds(const Tvg_Paint* paint, float* x, float* y, float* w, float* h);
 
 /*!
-* \fn TVG_EXPORT Tvg_Result tvg_paint_set_composite_method(const Tvg_Paint* paint, Tvg_Composite_Method method)
+* \fn TVG_EXPORT Tvg_Result tvg_paint_set_composite_method(const Tvg_Paint* paint, const Tvg_Paint* mask, Tvg_Composite_Method method)
```

Comment:
--const

Reply:
After you ask your question about getter, I realized that in fact I dont need it.
Removed.